frontend/src/App.jsx

```diff
@@ -1,26 +1,26 @@
 import {useState} from 'react';
-import logo from './assets/images/logo-universal.png';
-import './App.css';
-import {Greet} from "../wailsjs/go/main/App.js";
+import './App.scss';
+import {ProcessPullRequest} from "../wailsjs/go/main/App.js";
+import DiffViewer from "./components/DiffViewer/DiffViewer.jsx";
 
 function App() {
-    const [resultText, setResultText] = useState("Please enter your name below 👇");
-    const [name, setName] = useState('');
+    const [resultText, setResultText] = useState('');
+    const [url, setName] = useState('');
```

Comment:
setUrl?

Reply:
Oops, yes it should.  Good catch.

Reply:
Fixed and squashed.  I've also updated the screenshot after merging in the PR that adds tabs.